src/e2e/pages/dashBoardPage.ts

```diff
@@ -92,13 +97,14 @@ export class DashboardPage {
       name: "About what we fixed",
     });
     this.popupOkButton = page.getByRole("button", { name: "OK" });
+    this.popupCloseButton = page.getByRole("button", { name: "Close" });
 
     // chartTooltip
     this.chartTooltip = page.locator(
       '//div[starts-with(@class, "DashboardTopBanner_chart")]//button',
     );
     this.aboutActiveExposuresPopup = page.getByRole("dialog", {
-      name: "About your number of active exposures",
+      name: "About your number of fixed exposures",
```

Comment:
This doesn't seem right? Or at least, the variable name (`aboutActiveExposuresPopup`) implies that this isn't right.

Reply:
good point need to rename this

Reply:
Hmm OK, I'm still confused though why a test that previously expected the active tab to be visible, now all of a sudden expects the fixed tab to be visible, without any other changes?